Website/DesktopModules/Hotcakes/Core/Controllers/CheckoutController.cs

```diff
@@ -276,6 +288,28 @@ public ActionResult ApplyRewardPointsChange(FormCollection form)
 
             return new PreJsonResult(Web.Json.ObjectToJson(result));
         }
+        
+        /// <summary>
+        /// Attach Payment method to Stripe Payment Intent
+        /// </summary>
+        /// <returns></returns>
+        [HccHttpPost]
+        public ActionResult AttachPaymentMethod()
```

Comment:
There are a lot of values here without any defensive coding. Think about your testing... Do you think that's okay? If you think yes, that's fine. It's just a question. :)